test/ja/ja_casual.test.ts

```diff
@@ -2,27 +2,90 @@ import * as chrono from "../../src/";
 import { testSingleCase } from "../test_util";
 
 test("Test - Single Expression", function () {
-    testSingleCase(chrono.ja, "今日感じたことを忘れずに", new Date(2012, 8 - 1, 10, 12), (result) => {
-        expect(result.index).toBe(0);
-        expect(result.text).toBe("今日");
+    function testToday(pattern: string) {
```

Comment:
Could you not use a function like this and repeat testSingleCase() for each case?

I know it's a lot of duplicated code, but we want to ensure the tests are straightforward. (e.g. if the expect fails we will know which line and case)

Reply:
> we want to ensure the tests are really straightforward

I absolutely agree with you. 
I'll make the tests like so and maybe with less assertions like you mentioned [here](https://github.com/wanasit/chrono/pull/511#discussion_r1161188815)